src/tasks/TaskHelper.ts

```diff
@@ -103,14 +103,14 @@ export function registerTaskProviders(ctx: ExtensionContext): void {
 }
 
 export function hasTask(taskLabel: string, folder: WorkspaceFolder): boolean {
-    const workspaceTasks = workspace.getConfiguration('tasks', folder.uri);

```

Comment:
Now we can just pass the folder instead of `folder.uri`. Same as line 113 below.